demisto_sdk/commands/common/content/content.py

```diff
@@ -68,8 +72,12 @@ def git() -> Repo | None:
             1. Should be called when cwd inside content repository.
         """
         try:
-            repo = Repo(Path.cwd(), search_parent_directories=True)
+            if content_path := os.getenv('DEMISTO_SDK_CONTENT_PATH'):
+                repo = Repo(content_path)
+            else:
+                repo = Repo(Path.cwd(), search_parent_directories=True)
         except InvalidGitRepositoryError:
+            logger.debug('Git repo was not found.')
```

Comment:
1. isn't it an error/warning?
2. does it replace the message we already print?

Reply:
This is just a debug message.
The logic stays the same (returning `None` and let the command decide what to do with it)